src/partials/_action.hbs

```diff
@@ -1,14 +1,14 @@
 <nav>
   <ul class="action">
-    <a href="" class="type--link action--link">
+    <a href="" class="type--link action--link__alt">
```

Comment:
Just as an FYI, you should put something like a `#` in the href. Some browses won't give you the link pointer on hover if this is empty.